### ISSUE TITLE
Could not establish Internet connection

### ISSUE THREAD

**Issue by i2qbske7kq** [allotropia/nextcloud_files_libreoffice_edit#17]
Hello. I am trying to edit the files and every time I click on edit with libreoffice, it opens libreoffice which says "Could not establish Internet connection to" my ip. I have tried reinstalling libreoffice and reinstalling nextcloud. It did not work and I still get that error. It used to work during the previous install of nextcloud, then one day it stopped working. I dont remember doing anything to change it. I saw someone else having the same problem in the youtube comments of a video. Is there any fix?

Libreoffice version: 7.3.4.2 x64 Windows
Nextcloudpi 23.0.6

**Comment by smehrbrodt** [comment]
Sorry I can't really help with debugging internet connection problems here - no idea what might be wrong.

**Comment by i2qbske7kq** [comment]
Hello! I have managed to fix the issue by going back to libreoffice 6.4.5.1 win 86_x64 msi. It seems to work and save now like it should. This was the first libreoffice install I have tried to get this to work. You might be able to get a newer one working.

Thank you and everyone who has worked on this app for making it and taking time to work on it for free. This is one of my most used apps and I am glad to be able to have it working again.
